client/src/components/game/lab2/Instructions/thirdInstructions.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import Button from '../components/header/buttons/button';
+import './secondaryInstructions.css';
+
+/*
+Page containing the instructions for the second popup the user sees
+this instructional popup covers color contrast
+*/
+const ThirdInstructions  = ({closePage, activatePopup, toWhiteBackground,
+  background, selectOption}) => {
+
+  let alreadyCalled = false;
+  if (!alreadyCalled) {
+    selectOption('Protanopia');
+  }
+
+  const changeColors = () => {
+    activatePopup();
+  }
+
+  if (background !== 'white') {
+    toWhiteBackground();
+  }
+
+  return (
+    <div className="instructionsContainer">
+      <p className='secondInstructionTitle'>Instructions -- Part Three</p>
+      <ul>
+        <li className='instructionsItem'>
+          As you can see, this game can be very difficult for users who are
+          colorblind. The game was not created in an accessible manner for these
+          users. This is due to the colors being in contrast with both the
+          background and the other circles.
+        </li>
+          <br /><br />
+
+        <li className='instructionsItem'>
+          You can change the colors with the
+          <strong style={{marginLeft: '3px'}}>
+            Update Colors
+          </strong>
+          button in the upper left corner of the next screen.
+        </li>
+          <br />
+        <li className='instructionsItem'>
+          You can use any online calculator
+          or a calculator you've created for your lab! In order to fix the
+          problem, the contrast must be above a ratio of <strong>7:1</strong>. <em>Black is
+          not an applicable option, as the entire system relies upon black
+          for text coloring (giving you zero contrast between the background
+          and text).</em>
+        </li>
+          <br /><br />
+        <h2>Did you know?</h2>
+        <li className='instructionsItem'>
+          Color contrast is a measurement of how much two colors differ from
+          one another. This measurement makes a huge difference to people color
+          vision deficiencies.
+        </li>
+      </ul>
+      <div className='center'>
```

Comment:
I think there could maybe be more space between the end of the paragraph and the "I'm Ready!" button
<img width="865" alt="Screen Shot 2021-03-08 at 5 43 09 PM" src="https://user-images.githubusercontent.com/54957497/110391714-e778ae80-8035-11eb-985e-a4dcb48351f5.png">